atomic_reactor/utils/operator.py

```diff
@@ -644,13 +676,21 @@ def set_related_images(self):
             old = by_name.setdefault(new.name, new)
             # Check for potential conflict (same name, different image)
             if new.image != old.image:
-                msg = ("{old.description}: {old.image} X {new.description}: {new.image}"
-                       .format(old=old, new=new))
-                conflicts.append(msg)
-
-        if conflicts:
-            raise RuntimeError("{} - Found conflicts when setting relatedImages:\n{}"
-                               .format(self.path, "\n".join(conflicts)))
+                description_conflict = conflicts_map.get(old, [])
+                description_conflict.append(new)
+                conflicts_map[old] = (description_conflict)
+
+        if conflicts_map:
+            error_message = ("{} - Conflicts found when setting relatedImages. "
+                             "All images should be identical for named "
+                             "references.\n".format(self.path))
+            for old_image, new_images in conflicts_map.items():
+                error_message += ("The following images were found for the named "
+                                  "reference \"{}\":\n".format(old_image.pullspec_reference))
```

Comment:
In order to try to increase the comprehension of pullspec conflicts, I attempted to add two properties to a pullspec

* `pullspec_type`: This corresponds to the subclass. It represents where the related image is referenced. For example, it may be relatedImage, container, annotation, image, ... 
* pullspec_reference: This corresponds to what the actual pullspec is that we are trying to insert into the related_image. It should be as close as possible to what a user actually inserted into the CSV in order to aid comprehension. Since annotations have additional "processing" being added into the `self.name`, I thought that it would be less confusing to use the `self._image_key` to highlight where the conflict is.

Reply:
@tkdchen; Do you have any further questions/feedback here?

Reply:
No. Sorry for late response.